### ISSUE TITLE
Highlighting areas when hovering over the area name in a list item

### ISSUE THREAD

**Issue by pkraker** [pkraker/Headstart#20]
Highlighting areas when hovering over the area name in a list item would be useful.

**Comment by Bubblbu** [comment]
I would actually prefer to emphasize the two areas Map vs List. If there was a visible border users would not be tempted to click the list in order to interact with the map.
I personally think that would be a cleaner solution than adding map interaction to the list area.

What do you think?

**Comment by tanteuschi** [comment]
I don't agree. There are things the list can do much better and there are things the vis is better at. We need to combine those two for a really great user experience. and of course that should work both ways. If I interact with the vis I can see how the list changes. etc. 
The clever placement of the vis on the left side will make users interact with the vis before they start using the list. But of course we need to find out if that's actually the case.

**Comment by Bubblbu** [comment]
It's really weird, but I think that my comment was meant for another issue... #19 I still remember that I thought about that in #19's context.

But yes! Totally agree that we need to exploit and combine the strenghts of list and viz!

**Comment by tanteuschi** [comment]
ok then ignore my comment ;)
